### DISCUSSION TITLE
Unstable streams

### DISCUSSION THREAD

**Discussion by riftdea-isj** [ant-media/Ant-Media-Server#5589] [Q&A] [answered]
Hi @yashtandon113  or @Mohit-3196 , is there a way antmedia can capture or shows what's the reason for streams to be unstable. Right now I have 250+ streams on my servers, lately for unknown reasons they are showing on and off in the media web console....

![image](https://github.com/ant-media/Ant-Media-Server/assets/132421851/ab85f3de-3e04-438f-b399-296852e83b07)

![image](https://github.com/ant-media/Ant-Media-Server/assets/132421851/323fefbe-9fa2-448b-b51f-7f1ab961aff5)

Their value doesn't make sense as shown above. I'm using AM v2.5.3 enterprise. RTMP stream.
There is no problem if I change to rtsp.

**Comment by tremzo** [comment]
Do you mean your live streams keep changing from broadcasting state to offline? How are you sending the RTMP are you using FFMPEG or some other encoder? You might want to consider upgrading in the first instance, we are now up to version 2.6.4 this might help resolve some issues for you.

**Reply by riftdea-isj** [reply]
@timantmedia Yes offline to online then offline again, my own encoder(there were never this problem for 1 month). version 2.6.4 have issue of can't pull RTMP stream.  #5537 in this discuss explained why can't.

**Reply by tremzo** [reply]
@riftdea-isj Ok there was an issue with RTMP as a stream source that should have been resolved. Are you confirming that you are still experiencing an issue with that? 

This is the PR that should have fixed it https://github.com/ant-media/Ant-Media-Server/pull/5531. If you fetch the latest version there should be no issue. Please upgrade to the latest version and let us know if you are still having a problem with the RTMP pull.

**Reply by riftdea-isj** [reply]
Sorry @timantmedia for the late replied, was busy with another project. 
I've tested with the Antmedia 2.6.4 Enterprise, it still unable to pull RTMP stream while AM 2.5.3 is able to pull the RTMP stream.

**Reply by yashtandon113** [reply, marked as answer]
Hi @riftdea-isj 
The PR for RTMP pull fix is merged in a upcoming version 2.7.0. For now, you can download the latest snapshot from your account on antmedia.io.
You can check the change logs [**here**](https://antmedia.io/changelog/changelog.html).
